lib/sprockets/erb_processor.rb

```diff
@@ -1,31 +1,32 @@
 # frozen_string_literal: true
 require 'erb'
 
-module Sprockets
-  class ERBProcessor
-    # Public: Return singleton instance with default options.
-    #
-    # Returns ERBProcessor object.
-    def self.instance
-      @instance ||= new
-    end
+class Sprockets::ERBProcessor
```

Comment:
I was going to, but it gets loaded from `sprockets.rb`, inside (and thus after) the main `module Sprockets`.

As this file doesn't strike me as one people are likely to be requiring directly, it didn't seem worth bothering.